earth_enterprise/rpms/opengee-server/snippets/post-install.sh

```diff
@@ -125,14 +125,11 @@ fix_postinstall_filepermissions()
     chown $GEAPACHEUSER:$GEGROUP $BASEINSTALLDIR_OPT/gehttpd/htdocs/.htaccess
     chown -R $GEAPACHEUSER:$GEGROUP $BASEINSTALLDIR_OPT/gehttpd/logs
 
-    # Publish Root
+    # Publish Root - note these are not recursive
```

Comment:
Technically this file works differently than opengee-fusion/snippets/post-install.sh. For fusion, we check the permissions of the top level asset root directory and only run chown if those permissions are wrong. For server, we never run chown. I'm not sure that it matters a lot but it is somewhat inconsistent. Personally I prefer the server method - don't run a chown at all and let the user make the decision to do that. We'll want to make sure we clearly state somewhere that if your asset root or publish root permissions are wrong you'll need to run the appropriate command with the chown option.

I'll leave it up to you whether you want to address this or not. It would be nice to have them consistent but I can live without it.

Reply:
Yeah, I agree.  I'll remove $DOCHOWN and update the message here as well.